src/model/fxQuotes.js

```diff
@@ -287,44 +311,98 @@ class FxQuotesModel {
   }
 
   /**
-   * Handles an fxQuote response i.e PUT /fxQuotes/{id} requests
+   * Logic for handling fxQuote update requests e.g. PUT /fxQuotes/{id} requests
    *
-   * @param {object} headers
-   * @param {string} conversionRequestId
-   * @param {object} fxQuoteUpdateRequest
-   * @param {object} span
    * @returns {undefined}
    */
-  async handleFxQuoteUpdate (headers, conversionRequestId, fxQuoteUpdateRequest, span) {
+  async handleFxQuoteUpdate (headers, conversionRequestId, fxQuoteUpdateRequest, span, originalPayload) {
     const histTimer = Metrics.getHistogram(
       'model_fxquote',
       'handleFxQuoteUpdate - Metrics for fx quote model',
       ['success', 'queryName']
     ).startTimer()
 
+    let txn
     const fspiopSource = headers[ENUM.Http.Headers.FSPIOP.SOURCE]
     const childSpan = span.getChild('qs_fxQuote_forwardFxQuoteUpdate')
-    let txn
 
     try {
       await childSpan.audit({ headers, params: { conversionRequestId }, payload: fxQuoteUpdateRequest }, EventSdk.AuditEventAction.start)
-
-      this.validateHeaders(headers, conversionRequestId)
```

Comment:
any particular reason for replacing with inline code?

Reply:
I wasn't the one who worked on this but if I had to guess is that abstracting the code into a function wasn't any cleaner since the code is only needed for one occurance, so they did away with the function and just moved it into `handleFxQuoteUpdate` itself.

Reply:
ok